src/devices/Seesaw/SeesawBase.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Threading.Tasks;
+using System.Device.I2c;
+
+namespace Iot.Device.Seesaw
+{
+    public partial class Seesaw : IDisposable
+    {
+        private const byte Sessaw_Hardware_Id = 0x55;
```

Comment:
```suggestion
        private const byte Seesaw_Hardware_Id = 0x55;
```

Reply:
btw if this is I2c address then it should be called `DefaultI2cAddress` to follow other devices

Reply:
It's not an I2C address it a hardware identifier so that you can be sure you are talking to a Seesaw device. Underscores removed